src/tab-group.ts

```diff
@@ -311,49 +331,70 @@ export default class TabGroup extends LitElement {
     }
   }
 
-  #onSelectedTabIndicatorTransition() {
-    this.#setOverflowButtonsState();
-  }
-
-  #onTabListFocusout() {
-    // Set the last selected tab as tabbable so that when pressing Shift + Tab on the
-    // Tab Panel focus goes back to the last selected tab.
-    for (const [, tabElement] of this.#tabElements.entries()) {
-      tabElement.tabIndex = tabElement === this.#lastSelectedTab ? 0 : -1;
-    }
-  }
-
-  #onTabListResize() {
+  #onTabContainerResize() {
```

Comment:
This is simplified now - we only need to concern ourselves with the overflow buttons on resize.  Not updating the selected tab indicator.